common/src/main/java/com/wynntils/features/utilities/WeeklyConfigBackupFeature.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.utilities;
+
+import com.wynntils.core.WynntilsMod;
+import com.wynntils.core.components.Managers;
+import com.wynntils.core.consumers.features.Feature;
+import com.wynntils.core.persisted.Persisted;
+import com.wynntils.core.persisted.config.Category;
+import com.wynntils.core.persisted.config.ConfigCategory;
+import com.wynntils.core.persisted.storage.Storage;
+import com.wynntils.mc.event.TitleScreenInitEvent;
+import com.wynntils.utils.FileUtils;
+import java.io.File;
+import java.util.concurrent.TimeUnit;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+@ConfigCategory(Category.UTILITIES)
```

Comment:
Maybe the Wynntils category?

Reply:
Sure, although it fits both.

Reply:
magicus did, we mostly need UI refactors for that.